scripts/17--html--forms/README.md

```diff
@@ -92,50 +118,56 @@
     </select>
     ```
 
-    - also `<optgroup>` for grouping options
-    - more advanced fake select boxes using JS
+- also `<optgroup>` for grouping options
+- more advanced fake select boxes using JS
 
-- Text Area
+#### Textarea tag
 
-    ```html
-    <label for="message">Textarea</label>
-    <textarea id="message" name="message"></textarea>
-    ```
+- `<textarea>`
+- not self-closing
+- can enter multi-line text for eg. a comment
+- uses all the same attributes as `<input>`
+- however `value` goes *between* tags
+- careful: *all* whitespace between tags is included
 
-    - "value"
-        - goes *between* tags
-        - careful: *all* whitespace between tags is included
+```html
+<label for="message">Text Area</label>
+<textarea id="message" name="message"></textarea>
+```
 
-- Fieldset & Legends
-    - For grouping inputs that are related
+#### Fieldset & Legends
 
-    ```html
-    <fieldset>
-      <legend>About You</legend>
+- `<fieldset>` and `<legend>`
+- For grouping inputs that are related
+- Great for custom styling and for disabling a group of inputs at once
+- do not have to be nested inside the form element if `form` attribute is used
 
-      <label for="first-name">First Name</label>
-      <input type="text" name="first-name" id="first-name" />
+```html
+<fieldset>
+    <legend>About You</legend>
 
-      <label for="last-name">Last Name</label>
-      <input type="text" name="last-name" id="last-name" />
+    <label for="first-name">First Name</label>
+    <input type="text" name="first-name" id="first-name" />
 
-      <!-- more related inputs -->
-    </fieldset>
+    <label for="last-name">Last Name</label>
+    <input type="text" name="last-name" id="last-name" />
 
-    <fieldset>
-      <legend>Your Purchases</legend>
+    <!-- more related inputs -->
+</fieldset>
 
-      <!-- related inputs -->
-    </fieldset>
-    ```
+<fieldset>
+    <legend>Your Purchases</legend>
+
+    <!-- related inputs -->
+</fieldset>
+```
+
+#### Submitting
 
-- Submitting
-    - `<button type="...">`
-        - `submit`: submits forms
-        - `reset`: clears form (generally not that useful)
-        - `button`: creates a button that doesn't do anything - text is the `value` attribute
+You can submit using either a `<button>` or an `<input>`
```

Comment:
I feel like people use `<button>` more nowadays – but maybe not. I suspect someone will ask which one to prefer.